src/mame/atari/atari400.cpp

```diff
@@ -2010,11 +2108,8 @@ MACHINE_START_MEMBER( a400_state, a400 )
 }
 
 
-MACHINE_START_MEMBER( a400_state, a800 )
+void a800_state::machine_start()
 {
-	setup_ram(0, m_ram->size());
-	setup_ram(1, m_ram->size());
-	setup_ram(2, m_ram->size());
 	setup_cart(m_cartleft);
 	setup_cart(m_cartright);
```

Comment:
Machine start members that don’t call the base member are a red flag.  Can you structure things better so the derived classes can call the base member and then just do their additions?

Reply:
It's WIP but sure: this specific config should just call `setup_cart(m_cartright);`